sagemaker_batch_transform/pytorch_flores_batch_transform/docker/dockerd-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Copyright Facebook?

Reply:
Confirmed that it's acceptable to remove these lines given the background context.